birdhouse/default.env

```diff
@@ -11,7 +11,7 @@ export JUPYTERHUB_IMAGE_SELECTION_NAMES="pavics"
 
 export FINCH_IMAGE="birdhouse/finch:version-0.7.7"
 
-export THREDDS_IMAGE="unidata/thredds-docker:4.6.15"
+export THREDDS_IMAGE="unidata/thredds-docker:4.6.18"
```

Comment:
A note referring to the changlog fix should be added to indicate why `>=4.6.18` is strongly recommended.

Reply:
Or `>=5.2` (https://github.com/Unidata/tds/releases/tag/v5.2)

Reply:
> A note referring to the changlog fix should be added to indicate why `>=4.6.18` is strongly recommended.

Not sure I understand what to do.  Our own changelog CHANGE.md shows in the `docker run` command that new Thredds has newer log4j 2.15 and the affected version was 2.0 to 2.14.1 so implies 2.15 is okay.  I am not sure what else need to be clarified here?

Reply:
https://github.com/bird-house/birdhouse-deploy/pull/228/commits/49ac1b2e37fc80834ea5398657542cd7dd98b80e ?